webui/react/src/components/ThemeToggle.settings.ts

```diff
@@ -0,0 +1,23 @@
+import { BaseType, SettingsConfig } from 'hooks/useSettings';
+
+export interface Settings {
+  theme: string;
+}
+
+export enum ThemeClass {
+  SYSTEM = 'system',
+  LIGHT = 'light',
+  DARK = 'dark'
+}
```

Comment:
**nit:** this is the same idea as the `Mode` enum I mentioned throughout the review, but it might be better to have this in `useTheme` to better encapsulate theme changing code into a more central place. Idea of having this code in a shareable hook instead of a UI component.